lambda/custom-collection/README.md

```diff
@@ -0,0 +1,86 @@
+# MetaKeep Lambda - Custom ERC721
+
+This repository contains a Custom ERC721 Application that you can use as the starting point
+for using MetaKeep's Lambda Infrastructure.
+
+This project is intended to be used with the
+[MetaKeep Lambda Creation and Invocation Tutorial Series](https://docs.metakeep.xyz/docs/create-your-first-lambda), but you should be
```

Comment:
update the link. It should point to custom erc721 link

Reply:
I will update this once the link is added to documentation

Reply:
You can make it `custom-erc721`. I will use the same link